Automattic/newspack-newsletters
# Add Posts Inserter block

### All Submissions:






### Changes proposed in this Pull Request:



Adds a "Posts Inserter" block. It's in large part mimicking the core "Latest Posts" block, but adds the _preview, then edit_ flow. 

Closes #18 

### How to test the changes in this Pull Request:

1. Add new Newsletter
2. Add a "Posts Inserter" block
3. Observe a block with a latest posts list preview and an "Insert posts" button
1. Tweak the block settings in the right hand panel, observe the preview updating
1. Click the "Insert posts" button, observe that the previewed content is now in newsletter content
1. Edit the post titles and other elements
1. Send a test email/campaign. Observe that the email matches the editor content

### Other information:
